services/ssp-b/src/public/js/video-ad-tag.js

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+(async () => {
+  const ins = document.querySelector('ins.ads');
+  const script = document.querySelector('.ssp_tag');
```

Comment:
you called it ssp_b_tag in ad_tag.js

Reply:
are you ok keeping ".ssp_tag" ?

Reply:
resolving it since this code is unused and will not be touched until the video ad PR. if i deem this file unnecessary, i'll remove this file when i work on the video ad code today.